src/NzbDrone.Common/Disk/DiskTransferService.cs

```diff
@@ -511,6 +502,21 @@ private void TryMoveFileVerified(string sourcePath, string targetPath, long orig
             }
         }
 
+        private void VerifyFile(string sourcePath, string targetPath, long originalSize, string action)
+        {
+            var targetSize = _diskProvider.GetFileSize(targetPath);
+            if (targetSize != originalSize)
+            {
+                _logger.Info("File {0} incomplete, waiting in case filesystem is not synchronized. [{1}] was {2} bytes long instead of the expected {3}.", action, targetPath, targetSize, originalSize));
```

Comment:
discussion comment: should this be debug instead of info?

Reply:
I think Debug is more appropriate here since we're throwing if it's still an issue after the wait. I don't think we need to log how long it's being delayed for though.

```suggestion
                _logger.Debug("File {0} incomplete, waiting in case filesystem is not synchronized. [{1}] was {2} bytes long instead of the expected {3}.", action, targetPath, targetSize, originalSize));
```